### ISSUE TITLE
Full catalog link on the front page is broken (Safari)

### ISSUE THREAD

**Issue by gonsie** [LLNL/llnl.github.io#330]
clicking it takes you to the `https://software.llnl.gov/category/#/BUILDTOOLS` page... where none of the side bar buttons work and the search is broken/limited to the build-tools category. 

also, why is it javascript and not a regular link?

**Comment by IanLee1521** [comment]
hmm, using that link above seemed to work for me?

![image](https://user-images.githubusercontent.com/828452/76550580-98faf900-644f-11ea-8b33-7b2a4e7c2508.png)

**Comment by LRWeber** [comment]
The front page link takes me to https://software.llnl.gov/category/#/ALLSOFTWARE

**Comment by gonsie** [comment]
it seems to be a trickier bug. Once you go to the catalog (either through the full catalog link or by clicking any of the categories), the buttons on the left don't change your view. if you use the back button, and try to go to the catalog again, you are stuck with the subset you originally had.

Reproducer of my issue:
1. go to https://software.llnl.gov
2. click on the 'Build Tools' box
3. BUG: tabs on the left don't work.
4. hit the back button to go to the home page
5. click 'full catalog' link
6. BUG: you are still in the 'build tools' view.

**Comment by LRWeber** [comment]
Still all works fine for me.... Sounds like a browser (or script blocker?) issue.

**Comment by gonsie** [comment]
seems like this is a safari only issue.

**Comment by IanLee1521** [comment]
@gonsie - is this still an issue?

**Comment by gonsie** [comment]
Yep, still and issue (just with safari).

**Comment by SteVwonder** [comment]
Just chiming in to say that (on safari), I cannot right-click the link in question to open it in a new tab/window.  I assume switching it from javascript to a regular link would fix that?

**Comment by hauten** [comment]
Sadly not fixed by https://github.com/LLNL/llnl.github.io/pull/552. I can recreate the problem in Safari on my phone. I will take a closer look.

**Comment by hauten** [comment]
Well, my changes in https://github.com/LLNL/llnl.github.io/pull/554 (to change js link to regular, if I did it correctly) didn't solve this problem. A new side effect of that PR is that the hamburger links I've visited change to blue, so I kind of know what I was looking at before I clicked Back. But obv not a meaningful resolution to this bug. :/
![image](https://user-images.githubusercontent.com/31740953/157705204-ff4d14ed-62bb-4da9-96b3-eb2e136c1591.png)
